GOVERNACE.md

```diff
@@ -1,45 +1,75 @@
+This is a Work in Progress. Stay tuned for more follow-up updates! 
```

Comment:
Could you also modify https://github.com/tikv/community/blob/master/GOVERNACE-zh_CN.md in this PR?

Reply:
Will do later, after this one is submitted